coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -488,6 +488,11 @@ def flatten(context, node):
     else:
         end = end_val
 
+    if start_val < 0:
```

Comment:
Thanks for catching this. 

May be, one liner: `start = len(dims) + start if start < 0 else start`, but either is good.

Reply:
Actually, can you please add a small unit test that fails without this change in `test_torch_ops.py`?